<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MappingGeneratorDialog</class>
 <widget class="QDialog" name="MappingGeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="buttonDesc">
         <property name="maximumSize">
          <size>
           <width>157</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDesc">
         <property name="text">
          <string>Description File</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QScrollArea" name="scrollDesc">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>274</width>
            <height>288</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTreeView" name="treeView"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>157</width>
            <height>288</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableView" name="listAtributes">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="buttonAddClass">
         <property name="text">
          <string>Add Class</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonAddProperty">
         <property name="text">
          <string>Add Property</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonAddAsociation">
         <property name="text">
          <string>Add Asociation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="buttonOut">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>157</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelOut">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MappingGeneratorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MappingGeneratorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDesc</sender>
   <signal>clicked()</signal>
   <receiver>MappingGeneratorDialog</receiver>
   <slot>chooseDesc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOut</sender>
   <signal>clicked()</signal>
   <receiver>MappingGeneratorDialog</receiver>
   <slot>chooseOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAddClass</sender>
   <signal>clicked()</signal>
   <receiver>MappingGeneratorDialog</receiver>
   <slot>addClass()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>MappingGeneratorDialog</receiver>
   <slot>newSelection(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chooseDesc()</slot>
  <slot>chooseOut()</slot>
  <slot>newSelection(QModelIndex)</slot>
  <slot>addClass()</slot>
 </slots>
</ui>
